cgi-bin/DW/Media.pm

```diff
@@ -110,9 +110,15 @@ sub upload_media {
 
     # set the security
     my $sec = $opts{security} || 'public';
+    $sec = 'usemask' if $sec =~ /^(?:friends|access)$/;
```

Comment:
I did just change it to be explicit - all I was saying was that I thought it better to keep the other check in as well.